src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/ContactDiaryStore.swift

```diff
@@ -1004,7 +1004,12 @@ class ContactDiaryStore: DiaryStoring, DiaryProviding, SecureSQLStore {
 			return .failure(dbError(from: database))
 		}
 
-		return .success(contactPersons)
+		let arrangedContactPersons = contactPersons.sorted(by: {
+			$0.name.lowercased().folding(options: .diacriticInsensitive, locale: .current)
+				<
+				$1.name.lowercased().folding(options: .diacriticInsensitive, locale: .current)
+		})
```

Comment:
```suggestion
		let sortedContactPersons = contactPersons.sorted { $0.name.localizedCompare($1.name) == .orderedAscending }
```

I think that's a bit easier to read